rollup.config.js

```diff
@@ -0,0 +1,30 @@
+import commonjs from "@rollup/plugin-commonjs"
+import peerDepsExternal from "rollup-plugin-peer-deps-external"
+import postcss from 'rollup-plugin-postcss'
+import resolve from "@rollup/plugin-node-resolve"
+import vue from "rollup-plugin-vue"
+
+import packageJson from "./package.json"
+
+export default {
+  input: "src/index.js",
+  output: [
+    {
+      format: "cjs",
+      file: packageJson.main,
+      sourcemap: true
+    },
+    {
+      format: "esm",
+      file: packageJson.module,
+      sourcemap: true
+    }
+  ],
+  plugins: [
+    peerDepsExternal(), 
+    resolve(),
+    vue(), 
+    postcss(),
+    commonjs(),
```

Comment:
No babel? We don't transpile dependencies in our apps so we should do it here (ref https://github.com/nextcloud/calendar-js/blob/e21fadd5af9dddae2d997f7707395e7ff27a2406/rollup.config.js#L29).

Add our browserslist config to `package.json` as well:
- Dependency: https://github.com/nextcloud/calendar-js/blob/e21fadd5af9dddae2d997f7707395e7ff27a2406/package.json#L44
- Config in `package.json`: https://github.com/nextcloud/calendar-js/blob/e21fadd5af9dddae2d997f7707395e7ff27a2406/package.json#L96

Reply:
Good point. Will add that!